src/components/Navigation/Buttons/BackButton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Button from "./Button";
+import classes from "./Buttons.module.scss"
```

Comment:
U are mixing SCSS and styled components, it might be a good idea to choose smth one.